CMakeLists.txt

```diff
@@ -61,13 +61,13 @@ option(THEROCK_BUNDLE_SYSDEPS "Builds bundled system deps for portable builds in
 # Source settings.
 # Source can be consumed from the https://github.com/ROCm/rocm-libraries superrepo,
 # while the default is to consume the submodules defined in TheRock's `.gitmodules`.
-option(THEROCK_USE_ROCM_LIBRARIES "Use the `rocm-libraries` superrepo" OFF)
-if(THEROCK_USE_ROCM_LIBRARIES AND NOT THEROCK_ROCM_LIBRARIES_SOURCE_DIR)
-  message(FATAL_ERROR "If THEROCK_USE_ROCM_LIBRARIES is set, THEROCK_ROCM_LIBRARIES_SOURCE_DIR is required!")
-endif()
+option(THEROCK_USE_ROCM_LIBRARIES "Use the `rocm-libraries` superrepo" ON)
+set(THEROCK_ROCM_LIBRARIES_SOURCE_DIR_DEFAULT "${THEROCK_SOURCE_DIR}/rocm-libraries")
+set(THEROCK_ROCM_LIBRARIES_SOURCE_DIR "${THEROCK_ROCM_LIBRARIES_SOURCE_DIR_DEFAULT}" CACHE STRING "Path to rocm-libraries superrepo")
 
-if(THEROCK_ROCM_LIBRARIES_SOURCE_DIR)
-  cmake_path(ABSOLUTE_PATH THEROCK_ROCM_LIBRARIES_SOURCE_DIR NORMALIZE)
+cmake_path(ABSOLUTE_PATH THEROCK_ROCM_LIBRARIES_SOURCE_DIR NORMALIZE)
+if(NOT THEROCK_USE_ROCM_LIBRARIES)
+  message(WARNING "Building from indvidual submodules is deprecated. THEROCK_USE_ROCM_LIBRARIES should be set to `ON`.")
 endif()
```

Comment:
Is `-DTHEROCK_USE_ROCM_LIBRARIES=OFF` expected to work at all after this change? Maybe
* This should be a `FATAL_ERROR`
* The option should be set to `ON` with no option to change it
* We should remove the option entirely

What tripped me up a bit when testing locally was having the option cached to `OFF` from a prior build.